### PR TITLE
Initial commit with MPE Skeleton

### PR DESCRIPTION
**Pull request by prode13**
Initial commit with MPE Skeleton

### CODE REVIEW

File: repository.yaml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 digital-service: Guaranteed Minimum Pension
 repoVisibility: private_12E5349CFB8BBA30AF464C24760B70343C0EAE9E9BD99156345DD0852C2E0F6F
```
Comment (dione-obiorah) [inline repository.yaml:2]: Our services are public does the UI tests need to be public too?

File: src/test/resources/features/AcceptanceTests/ModularTests/Dashboard.feature [hunk 2]
```diff
@@ -5,4 +5,4 @@ Feature:As a PSA/PSP
   Scenario: Enter member details and navigate to MPE Results page
     Given I have a new session
     And I fill in the auth details for enrolment PSA with value A2100001
-    Then  I should be on the "Members Protection Enhancement (MPE) checker" page
\ No newline at end of file
+    Then  I should be on the "Members Protection Enhancement (MPE) checker" page
```
Comment (dione-obiorah) [inline src/test/resources/features/AcceptanceTests/ModularTests/Dashboard.feature:8]: It should be "Members Protections Enhancements

**Review by dione-obiorah — COMMENTED** [review]
There is only tests for PSA, can you add for PSP as well please

File: repository.yaml (hunk 1 above)
Comment (prode13) [reply repository.yaml]: Changed the repoVisibility to Public

File: src/test/resources/features/AcceptanceTests/ModularTests/Dashboard.feature (hunk 2 above)
Comment (prode13) [reply src/test/resources/features/AcceptanceTests/ModularTests/Dashboard.feature]: Changed to Members Protection Enhancement

File: repository.yaml [hunk 3]
```diff
@@ -1,3 +1,3 @@
-digital-service: Guaranteed Minimum Pension
-repoVisibility: private_12E5349CFB8BBA30AF464C24760B70343C0EAE9E9BD99156345DD0852C2E0F6F
+digital-service: Members Protection Enhancement
```
[the remark this answers is not in the document]
Comment (prode13) [reply repository.yaml]: Updated to 'Members Protections Enhancements'

File: src/test/resources/features/AcceptanceTests/ModularTests/Dashboard.feature [hunk 4]
```diff
@@ -2,7 +2,7 @@
 Feature:As a PSA/PSP
   I want to login and navigate to MPE Dashboard page
 
-  Scenario: Enter member details and navigate to MPE Results page
+  Scenario: Enter member details and navigate to MPE Results page for PSA
     Given I have a new session
     And I fill in the auth details for enrolment PSA with value A2100001
-    Then  I should be on the "Members Protection Enhancement (MPE) checker" page
+    Then  I should be on the "Members Protection Enhancements" page
```
[the remark this answers is not in the document]
Comment (prode13) [reply src/test/resources/features/AcceptanceTests/ModularTests/Dashboard.feature]: Updated to 'Members Protections Enhancements'

File: src/test/resources/features/E2ETest/E2E.feature [hunk 5]
```diff
@@ -1,13 +1,13 @@
 @regression @accessibility @e2e
-Feature:As a PSA/PSP
+Feature:As a PSA
   I want to view data, submit returns and file report for pension schemes.
 
   Scenario: Enter member details and navigate to MPE Results page
     Given I have a new session
     And I fill in the auth details for enrolment PSA with value A2100001
-    Then  I should be on the "Members Protection Enhancement (MPE) checker" page
+    Then  I should be on the "Members Protection Enhancements" page
```
[the remark this answers is not in the document]
Comment (prode13) [reply src/test/resources/features/E2ETest/E2E.feature]: Updated to 'Members Protections Enhancements'

**Review by dione-obiorah — APPROVED** [review]

**Review by kuldeepwadhwa9 — APPROVED** [review]

**Review by dione-obiorah — APPROVED** [review]

**Pull request merged**